lib/active_triples/persistence_strategies/parent_strategy.rb

```diff
@@ -30,7 +30,21 @@ def persisted?
     end
 
     ##
-    # Destroys the resource by removing it graph and references from the 
+    # Mark that this resource has not yet been loaded.
+    def not_loaded
```

Comment:
:+1: mutators [should end in bangs](http://stackoverflow.com/questions/612189/why-are-exclamation-marks-used-in-ruby-methods).